.gitignore

```diff
@@ -14,6 +15,7 @@ doc/html
 doc/*.tag
 doc/*.inc
 doc/doxygen.conf
+doc/xml
```

Comment:
I should add that. Oddly enough when I run py.test I don't see this; I'm not sure why